schema/filter/strategy/impl/DefaultFilterStrategy.go

```diff
@@ -0,0 +1,63 @@
+package impl
+
+import (
+	mqFilter "github.com/th2-net/th2-common-go/schema/queue/configuration"
+	p_buff "th2-grpc/th2_grpc_common"
+)
+
+type DefaultFilterStrategy struct {
+	extractFields Th2MsgFieldExtraction
+}
+
+func (fs *DefaultFilterStrategy) Construct() {
+	fs.extractFields = Th2MsgFieldExtraction{}
+}
+
+func (fs *DefaultFilterStrategy) Verify(messages *p_buff.MessageGroup, filters []mqFilter.MqRouterFilterConfiguration) bool {
+	// checking
+	for _, filter := range filters {
+		for _, anyMessage := range messages.Messages {
+			if fs.CheckValues(fs.extractFields.GetFields(anyMessage), filter.Metadata.([]mqFilter.FilterFieldsConfiguration)) {
```

Comment:
We can improve performance if filter extracts required value from metadata. Each filter: session_alias, message_type, direction and etc should have logic to extract value from message / raw message